index.html

```diff
@@ -116,18 +116,23 @@
 
 It's not an abitrary choice, and it's not one that was looked at lightly.
 - Globally accepted standard
-- Versatility of being distributed
+- Versatility of being distributed (DVCS)
 - Free & Open-Source!!
 - Enabling better development practices
 - Critical in taking further steps towards modern application development.
+- And finally, even Microsoft made the jump!!!
 
 There are tools to help the transition!
 
 ---
 
 ## About Git
 
----
+By far, the most widely used modern version control system in the world today is Git. Git is a mature, actively maintained open source project originally developed in 2005 by Linus Torvalds, the famous creator of the Linux operating system kernel.  
+
+Having a distributed architecture, Git is an example of a DVCS (hence Distributed Version Control System). Rather than have only one single place for the full version history of the software as is common in once-popular version control systems like CVS or Subversion (also known as SVN), in Git, every developer's working copy of the code is also a repository that can contain the full history of all changes.  
+
+In addition to being distributed, Git has been designed with performance, security and flexibility in mind.
 
```

Comment:
Need slide separater

```suggestion

---

```